bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -158,7 +161,33 @@ f_dest.attrs['fit_param'] = args.fit_param
 f_dest.attrs['log_param'] = args.log_param
 with h5py.File(args.template_file, "r") as f_src:
     f_src.copy(f_src["./"], f_dest["./"], "input_template_params")
-temp_samples = numpy.vstack((template_pars)).T
+
+
+# Rejection Sampling
+beta = args.mchirp_downsample_power
+power_law = lambda x: x**(beta)
+mchirp_index = args.fit_param.index('mchirp') # Index of mchirp in fit parameters
```

Comment:
I would remove the comment as it doesn't add anything compared to reading the code